assopy/models.py

```diff
@@ -841,6 +841,10 @@ class Invoice(models.Model):
 
     objects = InvoiceManager()
 
+    @property
+    def total(self):
+        return self.price + self.vat_in_local_currency
```

Comment:
not sure if that's correct. If i understand correctly `price` is in euros, and `vat_in_local_currency` is in local, in case of EP2018 in GBP. (should be in the same what local currency is set to)

Reply:
what do you suggest?